docs/SSL/lavalink-with-ssl.md

```diff
@@ -11,3 +11,11 @@ Port : 443
 Password : "www.freelavalink.ga"
 Secure : true
 ```
+Hosted by @ [](https://dsc.gg/extinctop)
```

Comment:
@ blank? https://github.com/MilanoDevelopment/lavalink-list/blob/patch-1/docs/SSL/lavalink-with-ssl.md you need to put something in the brackets lol